docker/pypi/wmagent-base/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM registry.cern.ch/cmsweb/oracle:21_5-stable as oracle
+FROM registry.cern.ch/cmsweb/dmwm-base:pypi-20230525
+MAINTAINER Valentin Kuznetsov vkuznet@gmail.com
+
+# directory where we place all the relevant scripts and configurations for WMAgent docker image
+ENV WMA_ROOT_DIR=/data
+
+# Install basic OS package dependencies
+RUN apt-get update
+RUN apt-get install -y libmariadb-dev-compat libmariadb-dev apache2-utils cron mariadb-server myproxy voms-clients rlwrap libaio1 && apt-get clean
+
+# Install some debugging tools
+RUN apt-get install -y hostname net-tools iputils-ping procps && apt-get clean
+
+# copy oracle client:
+COPY --from=oracle /usr/lib/oracle /usr/lib/oracle
+ENV LD_LIBRARY_PATH=/usr/lib/oracle
+ENV PATH=$PATH:/usr/lib/oracle
+ENV PKG_CONFIG_PATH=/usr/lib/oracle
+
+# Add all deployment needed directories, install and run scripts
+ADD etc $WMA_ROOT_DIR/etc
+ADD manage $WMA_ROOT_DIR/manage
```

Comment:
Isn't the `manage` script origin (as currently placed in the repository) `bin/manage` ? At least it used to be...  in the original PR. I can see you have moved it out of `bin` now. But it was there for a reason - see my comment about the way how  `bin/` and `etc/` should be distributed from both pypi and docker origins.

Reply:
I just replied in the main pull request on this. But can you please clarify why `manage` must be in the `bin` directory? It is a single script, that's why I moved it out. It will be taken care in the wmagent image though with the same logic that you implemented (other than contained in bin).